src/components/pages/RoleSignup/Applications/Mentor.js

```diff
@@ -72,444 +104,413 @@ const Mentor = ({ dispatch, error, successPage }) => {
-                    <Input placeholder="Current company" />
-                  </Form.Item>
-                </Col>
-                <Col md={20} xs={24}>
-                  <Form.Item
-                    label="Current position"
-                    type="text"
-                    name="current_position"
-                    value={formValues.current_position}
-                    onChange={handleChange}
-                    style={{ margin: '.5rem 1rem .5rem' }}
-                    rules={[
-                      {
-                        required: true,
-                        message: 'Current position is required.',
-                      },
-                    ]}
-                  >
-                    <Input placeholder="Current position" />
-                  </Form.Item>
-                </Col>
-              </Row>
+            <Divider />
 
-              <hr />
+            {/*Work*/}
+            <Row className="row-container">
+              {/*Company*/}
+              <Col md={20} xs={24}>
+                <Form.Item
+                  label="Current company"
+                  type="text"
+                  name="current_company"
+                  value={formValues.current_company}
+                  onChange={handleChange}
+                  rules={[
+                    {
+                      required: true,
+                      message: 'Current company is required.',
+                    },
+                  ]}
+                >
+                  <Input placeholder="Current company" />
+                </Form.Item>
+              </Col>
+              {/*Position*/}
+              <Col md={20} xs={24}>
+                <Form.Item
+                  label="Current position"
+                  type="text"
+                  name="current_position"
+                  value={formValues.current_position}
+                  onChange={handleChange}
+                  rules={[
+                    {
+                      required: true,
+                      message: 'Current position is required.',
+                    },
+                  ]}
+                >
+                  <Input placeholder="Current position" />
+                </Form.Item>
+              </Col>
+            </Row>
 
-              <Row style={{ padding: '3% 0 3% 3%' }}>
-                <Col className="test" md={22} xs={24}>
-                  <Form.Item
-                    className="mentorshipArea"
-                    label="What areas are you wanting to provide mentorship in?"
-                    tooltip={{
-                      title: 'What development role have you trained for?',
-                      icon: <InfoCircleOutlined />,
+            {/*CheckBoxes*/}
+            <Row>
+              <Col className="mentorshipArea" md={20} xs={24}>
+                <Form.Item
+                  size="medium"
+                  className="form-group"
+                  label="What areas are you wanting to provide mentorship in?"
```

Comment:
Admittedly tiny quibble here, but I think either "do you want" or "would you like" would be better for phrasing instead of "are you wanting" since they don't seem quite as passive.